File: 1045/1045.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var valores = lines[0].split(" ").map(Number);

valores.sort((x, y) => y - x);

var [a, b, c] = valores;

if (a >= b + c) {
  console.log("NAO FORMA TRIANGULO");
} else {
  if (a * a == b * b + c * c) {
    console.log("TRIANGULO RETANGULO");
  } else if (a * a > b * b + c * c) {
    console.log("TRIANGULO OBTUSANGULO");
  } else if (a * a < b * b + c * c) {
    console.log("TRIANGULO ACUTANGULO");
  }
  if (a == b && a == c && b == c) {
    console.log("TRIANGULO EQUILATERO");
  } else if (a == b || a == c || b == c) {
    console.log("TRIANGULO ISOSCELES");
  }
}

File: 1045/1045.py
valores = list(map(float, input().split()))

valores.sort(reverse=True)

a, b, c = valores

if a >= b + c:
    print("NAO FORMA TRIANGULO")
else:
    if a*a == (b*b) + (c*c):
        print("TRIANGULO RETANGULO")
    elif a*a > (b*b) + (c*c):
        print("TRIANGULO OBTUSANGULO")
    elif a*a < (b*b) + (c*c):
        print("TRIANGULO ACUTANGULO")
    if a == b and a == c and b == c:
        print("TRIANGULO EQUILATERO")
    elif a == b or a == c or b == c:
        print("TRIANGULO ISOSCELES")
